fpdf/encryption.py

```diff
@@ -218,22 +249,114 @@ def encrypt_bytes(self, data, obj_id):
             return self.encrypt_AES_cryptography(key, data)
         return ARC4().encrypt(key, data)
 
-    def encrypt_AES_cryptography(self, key, data):
-        iv = self.get_initialization_vector(16)
-        padder = PKCS7(self.key_length).padder()
+    def encrypt_AES_cryptography(self, key: bytes, data: bytes) -> bytes:
+        """Encrypts an array of bytes using AES algorithms (AES 128 or AES 256)"""
+        iv = bytearray(self.get_random_bytes(16))
+        padder = PKCS7(128).padder()
```

Comment:
Why hardcoding 128 there?
Shouldn't that match `self.key_length`, that is used in `CryptFilter`?

Reply:
All AES algorithms (either 128 or 256 bit key) have 16 bytes (128 bit) blocks - that's the size of the padding we must use.
https://www.cryptosys.net/manapi/api_blockciphersizes.html

Reply:
Alright then!